velox/docs/monthly-updates/may-2022.rst

```diff
@@ -0,0 +1,67 @@
+********************
+May 2022 Update
+********************
+
+Documentation
+=============
+
+* Add documentation for :doc:`SIMD <../develop/simd>`.
+* Add :doc:`Velox in 10 min <../velox-in-10-min>` tutorial.
+
+Core Library
+============
+
+* Add support for user-defined join operators.
+* Build abstract and portable SIMD using xsimd.
+* Add AlignedStlAllocator for folly F14 containers.
+* Add Substrait FilterNode and FieldAccessTypedExpr, CastTypedExpr support.
+* Add Decimal type.
+* Add Checkpointing for SsdCache.
+* Add SSE4.2 support.
+* Add TPC-H Connector.
+* Add support for constant partitioning columns.
+* Add fast path for const, flat encoded primitive arguments in SimpleFunctionAdapter.
+* Add support for more types in HivePartitionFunction.
+* Add support for Apple M1.
+* Add Velox to Substrait converter.
+* Add spilling to HashAggregation.
+* Add support for anti join with filter.
+
+Presto Functions
+================
+
+* Add :func:`histogram` function for integral, floating-point, boolean, timestamp, and date types.
+* Implement DISTINCT FROM.
+* Support TimestampWithTimezone in :func:`day_of_week`, :func:`day_of_year`, and :func:`year_of_week` functions.
```

Comment:
Thank you for writing up the monthly update! I think the support for TimestampWithTimezone in `minute`, `second`, and `millisecond` functions has also been added in May. (https://github.com/facebookincubator/velox/pull/1669)